source/solving_strategies/strategies/residual_based_picard_solver.py

```diff
@@ -1,4 +1,14 @@
-from source.solving_strategies.strategies.rasidual_based_solver import ResidualBasedSolver
+from source.solving_strategies.schemes.forward_euler1_scheme import ForwardEuler1
```

Comment:
Only import specific scheme where the constructor is called. Do not import everything at the beginning of the file.